### ISSUE TITLE
Change the property name in the MinimumWPVersionTrait trait

### ISSUE THREAD

**Issue by dingo-d** [WordPress/WordPress-Coding-Standards#2113]
## Is your feature request related to a problem?

In the `MinimumWPVersionTrait` trait, we have a public property that can be referenced in the sniffs, called `$minimum_supported_version`. However, the CLI argument for setting the minimum supported WP version is named `minimum_supported_wp_version`.

https://github.com/WordPress/WordPress-Coding-Standards/blob/9544c7e0e78fc85405b32076ec2949c2e59854d1/WordPress/Helpers/MinimumWPVersionTrait.php#L71

And that can cause some confusion when writing sniffs - in the tests and in the sniffs we are using one naming (`minimum_supported_version`), and when the property should be consumed, either via CLI, or via ruleset, we need to use the other name (`minimum_supported_wp_version`):

```xml
<config name="minimum_supported_wp_version" value="5.4"/>
```

or

```bash
phpcs --runtime-set minimum_supported_wp_version 4.5
```

## Describe the solution you'd like

The solution would be to change this property in the trait to be named `$minimum_supported_wp_version`. All the sniffs consuming this trait would be affected, and the tests as well, but since we would release this in the 3.0.0 version, this is an allowed BC break.

**Comment by jrfnl** [comment]
I fully support this proposal. What about naming it `minimum_wp_version` in that case ? I mean, I'm wondering if the `supported` in the property/CLI name has added value... ?

Historical perspective:
* IIRC the property was added first and as the property would be added to WP specific sniffs via the ruleset, the  minimum_supported_version` name was descriptive enough.
* At a later point in time, support for setting the property for all sniffs via the command-line was added.
    Now when using `minimum_supported_version` from the CLI without the context of the specific sniffs, it could be confusing... minimum supported version of **_what_** ? Which is why the CLI implementation uses a slightly different name `minimum_supported_wp_version`.

**Comment by GaryJones** [comment]
I'm happy with `minimum_wp_version`. Would need calling out in 3.0.0 changelogs/docs/wiki. Bit of a pain for consumers to update their PHPCS config files just to help us align a name internally, but the major release is the time to do it :-)

**Comment by dingo-d** [comment]
Yeah, `minimum_wp_version` sounds best. The aim is to make the naming of the cli argument and the trait property the same to have more consistency.

And since it's a major change we would denote what was changed in the changelog ofc 🙂

**Comment by jrfnl** [comment]
I'll prepare a PR for this to be pulled after PR #2112 has been merged (and will add information about the change to the WIP upgrade guides and such which I have locally).

**Comment by jrfnl** [comment]
🎗️ This will also need a change in the Wiki.
